### DISCUSSION TITLE
发一张我画的露西吧

### DISCUSSION THREAD

**Discussion by LiuLiBHZ** [share121/inter-knot#5288] [绝区零]
![微信图片_20240922111720](https://github.com/user-attachments/assets/b817e6b5-e150-4c9f-8337-a19b81b2d25d)

**Comment by Yizakl** [comment]
# 大佬！！！！！！！
我是fw

**Reply by LiuLiBHZ** [reply]
一般画画人2333

**Comment by keerdun** [comment]
好好好，不过是不是给加强了<img src="https://img-static.mihoyo.com/communityweb/upload/3b2e51180d866548d11c650a2dcf6d43.png" alt="三月七_暗中观察" width="75" >

**Reply by LiuLiBHZ** [reply]
狠狠地加强！新版本终于送露西了，我到现在都没抽到！绝了

**Comment by BJ-cmyk** [comment]
灌注了，我会永远视奸大佬您的
![1726838748273](https://github.com/user-attachments/assets/d3bbceea-e6a9-45a0-a176-ffc4f4132f75)

**Reply by violet-3351** [reply]
啊？？？？？有点………坏遗你的盛粪

**Comment by Letyem** [comment]
好看，就是一直抽不到<img src="https://img-static.mihoyo.com/communityweb/upload/9939adf214795d97dad9abd90265aab2.png" alt="帕姆_哭哭" width="75" >

**Reply by LiuLiBHZ** [reply]
确实，我也抽不到。不过新版本直接送

**Comment by manyshoes** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/03/21/fcba628ce15730cf0dbf480ec338f646_3898608417719823301.png" alt="丽都漫步-比利" width="75" >还以为是官图

**Reply by LiuLiBHZ** [reply]
谢谢！

**Comment by Muano2333** [comment]
大佬！

**Comment by hot-ice-seele** [comment]
豪堪

**Comment by Doubudou45** [comment]
史诗级加强ヽ(*´з｀*)ﾉ

**Comment by keyuanya** [comment]
佬大！
![c648ffd92573c1bbe143b84ca6b75abd7efa4a6d7965c6fb3a75761e0439e00f_upscayl_3x_remacri](https://github.com/user-attachments/assets/1cf377b8-1643-4b4f-91d8-bab824907551)

**Comment by fzdqz** [comment]
豪堪！谢谢佬

**Comment by miheihuyou** [comment]
为大佬献上膝盖

**Comment by 2390041242** [comment]
好看

**Comment by Lephaethon** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/06/19/8701a3b5fc90a6c37ba5491913b8218f_2209157026344411896.png" alt="布洛妮娅 开心鸭" width="75" >
**大佬！！！**

**Comment by 210flight** [comment]
### 佬!****

**Comment by LeoneMafia** [comment]
大佬好技术！！！

**Comment by d2531987182** [comment]
好看好看<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >

**Comment by CartersofBarbel** [comment]
太喜欢您画的金丝姬哀歌了
